Lib/test/test_time.py

```diff
@@ -124,9 +124,16 @@ def test_clock_monotonic(self):
     @unittest.skipUnless(hasattr(time, 'clock_gettime'),
                          'need time.clock_gettime()')
     def test_pthread_getcpuclockid(self):
+        from ctypes import c_void_p, sizeof
         clk_id = time.pthread_getcpuclockid(threading.get_ident())
         self.assertTrue(type(clk_id) is int)
-        self.assertNotEqual(clk_id, time.CLOCK_THREAD_CPUTIME_ID)
+        # when in 32-bit mode AIX only returns the predefined constant
+        if not sys.platform.startswith("aix"):
+            self.assertNotEqual(clk_id, time.CLOCK_THREAD_CPUTIME_ID)
+        elif ((sizeof(c_void_p) * 8) == 64):
```

Comment:
Is it possible to use `sys.maxsize` in this check to avoid introducing a dependency on `ctypes` into the test case?

Reply:
done. Thx. sys.maxsize.bitsize() is a new function for me!